admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,150 @@
+[[admin-guide-sched-pod-affinity]]
+= Using Pod Affinity and Anti-Affinity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+_Pod affinity_ and _pod anti-affinity_ allows you to specify rules about how pods should be placed relative to one another. For example, you could spread or pack pods within a service or relative to pods in other services; or, you can make sure a pod is scheduled on a machine with an SSD on it. 
+
+Pod affinity and anti-affinity allow you to constrain which nodes your pod is eligible to be scheduled on based on the labels on other pods. A xref:../../architecture/core_concepts/pods_and_services.html#labels[label] is a key/value pair.
+
+* Pod affinity tells the scheduler to locate a new pod on the same node as pods with the same labels.
+* Pod anti-affinity prevents the scheduler from locating a new pod on the same node as pods with the same labels.
```

Comment:
`same labels` does not sound correct. Basically it is determined based on the label selectors in rules part of input pod and the labels on the pods on nodes.